src/transformers/Streamable.js

```diff
@@ -0,0 +1,44 @@
+import { URL } from 'url';
+import fetch from 'node-fetch';
+
+const getTrimmedPathName = pathname => pathname.replace(/^\/|\/+$/g, '');
+
+export const getNormalizedStreamableUrl = url => {
+  const { pathname } = new URL(url);
+  const streamableEmbedUrl = 'https://api.streamable.com/oembed.json?url=';
+  const trimmedPathName = getTrimmedPathName(pathname).split('/');
+
+  if (trimmedPathName.length === 1) {
+    return `${streamableEmbedUrl}${url}`;
```

Comment:
The Streamable embed url shouldn’t be attached

```suggestion
    return url;
```